.env.example

```
@@ -27,4 +27,9 @@ REDIS_URL=redis://localhost:6379/0
 
 # NATS Configuration
 NATS_URL=nats://localhost:4222
-NATS_TOKEN=the-nats-token
\ No newline at end of file
+NATS_TOKEN=the-nats-token
+
+# JWT Configuration
+AUTH_SECRET_KEY: str = 'your_secret_key'
```

dotenv-linter (4.0.0)

[warning] [IncorrectDelimiter] The AUTH_SECRET_KEY: str  key has incorrect delimiter

(IncorrectDelimiter)